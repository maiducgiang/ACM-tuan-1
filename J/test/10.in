59
442,244(857)
504,891(138)
293,804(710)
710,609(213)
937,865(621)
346,571(143)
608,909(524)
764,759(639)
510,668(894)
846,312(942)
417,687(813)
996,577(379)
298,450(697)
660,249(584)
141,189(663)
505,598(735)
261,100(711)
645,421(157)
727,768(1000)
824,272(448)
294,985(348)
253,715(982)
578,278(752)
110,722(381)
478,171(833)
705,771(204)
687,698(240)
664,200(451)
130,423(913)
960,288(244)
336,708(605)
505,807(306)
527,329(613)
710,250(326)
354,907(897)
432,156(372)
322,341(809)
260,925(569)
124,167(479)
867,185(968)
157,665(398)
638,975(864)
365,635(728)
924,659(246)
437,651(560)
849,859(587)
299,334(517)
422,329(266)
713,847(131)
755,150(864)
697,747(196)
696,988(379)
595,245(903)
106,956(657)
140,511(843)
929,718(574)
812,376(459)
175,965(542)
355,375(426)
