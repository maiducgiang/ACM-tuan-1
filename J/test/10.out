441802613 999000
168128749 333000
146755453 499500
59158217 83250
8675257 9250
86556143 249750
121660123 199800
2122208 2775
85026371 166500
28182221 33300
69545021 166500
497790401 499500
298152247 999000
131917867 199800
23508079 166500
168364379 333000
28982179 111000
80596967 124875
909619154 1249875
102931022 124875
98230121 333000
253462267 999000
288850237 499500
36870553 333000
238846831 499500
235021811 333000
114501757 166500
663536251 999000
13029349 99900
239831989 249750
336371897 999000
56144611 111000
131700571 249750
177385019 249750
11818433 33300
17988509 41625
80504867 249750
65166161 249750
15505414 124875
866318783 999000
157507733 999000
23642107 37000
365270093 999000
307911529 333000
437213909 999000
106126216 124875
299035183 999000
421906937 999000
178283321 249750
13970291 18500
697049449 999000
696291391 999000
99108443 166500
11872189 111000
11697611 83250
116098607 124875
30057929 37000
175789577 999000
118340017 333000
